nixos-modules/microvm/options.nix

```diff
@@ -590,6 +590,12 @@ in
       description = "Custom CPU template passed to firecracker.";
     };
 
+    firecracker.enablePci = mkOption {
+      type = types.bool;
+      default = lib.versionOlder "1.13.0" pkgs.firecracker.version;
```

Comment:
Wait what, this is backwards. Why did this work for me?

Reply:
Lol, I used the wrong function , but I also got the args the wrong way around so the two bugs cancelled each other out.